packages/terra-show-hide/src/ShowHide.jsx

```diff
@@ -55,9 +81,45 @@ const ShowHide = (props) => {
     preview,
     intl,
     isOpen,
+    focusRef,
     ...customProps
   } = props;
 
+  const theme = React.useContext(ThemeContext);
+  const contentRef = React.useRef(null);
```

Comment:
You can import these methods like this:
import React, { useContext, useRef, useCallback } from 'react';

Of course you would add all the ones that you require.

Reply:
Is there any benefit of importing them that way rather than use like **React.useContext**?

Reply:
There is no programmatic advantage outside of not having to type "React" before every usage. As much as I prefer the old school way of typing the library name first, Charles' suggestion makes it consistent with the rest of Terra.

Reply:
I picked it up from theme context and kind of thought it is consistent with the rest of Terra the way it is: https://github.com/cerner/terra-core/blob/70ccf4fee8dafedd22ccdb4a224c2227bc7e764f/packages/terra-alert/src/Alert.jsx#L138, but I checked the rest of cases, and for useEffect the Charles's version is used everywhere, so changing for consistency: https://github.com/cerner/terra-core/pull/3838/commits/22a779db64ad198d7388aa83c1d11e1539b14033

Reply:
Terra and inconsistency, name a better duo.